viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -48,6 +51,44 @@
       projectCollection.getProjects();
     }
 
+    var filterByUser = function () {
+      var input = $('#userNameBox').val();
+      var rows = $('#project-list').find('tr');
+      if (input === "") {
+        rows.show();
+        return;
+      }
+      rows.hide();
+      var filteredRowIds = _.uniq(rows.filter(function () {
+        var $t = $(this);
+        return $t.find('#innerCreatedBy').find("label").text().toLowerCase().indexOf(input.toLowerCase()) !== -1;
+      }).map(function () {
+        var row = $(this);
+        return row.attr('id');
+      }));
+      rows.filter(function () {
+        var $t = $(this);
+        return _.contains(filteredRowIds, $t.attr('id'));
+      }).show();
+    };
+
+
+    var userFilterVisibility = function (showFilters) {
+      if (showFilters) {
```

Comment:
Why are you using css visibility you could only use function `hide` or `show` or `toggle` ?

Reply:
Because ccs visibility still reserves space in the dom, I want it to be fully removed from the DOM.

Reply:
Changed it.